src/altinn-app-frontend/src/sagas/index.ts

```diff
@@ -32,6 +33,7 @@ function* root(): SagaIterator {
   yield fork(FormDataModelSagas);
   yield fork(LanguageSagas);
   yield fork(TextResourcesSagas);
+  yield fork(AppLanguagesSagas)
```

Comment:
Is it possible to check the config for the language selector here instead?
That way the app won't need to have a watcher sitting around doing nothing.
(I'm not 100% sure what's available in a given time during runtime.)
Also if `AppLanguageSagas` are very different form `LanguageSagas`. on line 34, maybe they should be named more explicitly?
Maybe `LanguageSelectorSagas`? And could it maybe be moved into a branch inside of `LanguageSagas`?

The config check could also alternatively be done inside the watcher, to keep this file free from tests.
Is this

Reply:
Im not sure if this is possible. Since I want it to run it every time the Layout file changes. Becuase the Stateless app has different config file from the Statefull app, we need to listen to the change in config.

I am not that used to redux saga, so if you have suggestions for what could work I am open to it :P